BSP/STM32F413/Makefile

```diff
@@ -44,6 +44,7 @@ $(BSP_DIR)/Src/*.c	\
 $(BSP_DIR)/STM32F4xx_HAL_Driver/Src/*.c	\
 $(FREERTOS_DIR)/*.c	\
 $(UTIL_DIR)/Src/*.c	\
+$(BSP_DIR)/../Src/*.c	\
```

Comment:
why is this in this pr

Reply:
Needs to be there to compile the I2C.c file